src/content/fr/2019/mobile-web.md

```diff
@@ -0,0 +1,291 @@
+  <figcaption id="fig4-caption">Figure 4. Exemple de ce à quoi ressemble un texte dont le contraste des couleurs est insuffisant. Avec l’aimable autorisation de LookZook</figcaption>
+</figure>
+
+Pour des statistiques sur le daltonisme dans d’autres groupes démographiques, voir [ce document](https://web.archive.org/web/20180304115406/http://www.allpsych.uni-giessen.de/karl/colbook/sharpe.pdf).
+
+### Taille des polices
+
+La deuxième partie de la lisibilité consiste à s’assurer que le texte est suffisamment grand pour être lu facilement. C’est important pour tous les types d'utilisateurs, mais surtout pour les personnes âgées. Les tailles de police inférieures à 12&nbsp;px ont tendance à être plus difficiles à lire.
+
+Sur l’ensemble du web, nous avons constaté que 80,66&nbsp;% des pages web répondent à ce critère de référence.
+
+## Zoom, mise à l’échelle et rotation des pages
+
+### Zoom et mise à l’échelle
+
+Il est incroyablement difficile de concevoir un site qui fonctionne parfaitement sur des dizaines de milliers de tailles d’écran et de dispositifs. Certaines personnes ont besoin d’une taille de police plus importante pour lire, zoomer sur les images de vos produits, ou ont besoin d’un bouton plus grand parce qu’il est trop petit et a échappé à votre équipe d’assurance qualité. C’est pour de telles raisons que les fonctions de zoom et de mise à l’échelle des appareils sont si importantes&nbsp;; elles permettent aux gens de modifier nos pages pour qu’elles répondent à leurs besoins.
+
+Il existe de très rares cas où la désactivation est acceptable, par exemple lorsque la page en question est un jeu en ligne utilisant des commandes tactiles. Si cette option est laissée activée dans ce cas, les téléphones des joueurs feront un zoom avant et arrière chaque fois que le joueur tapera deux fois sur le jeu, ce qui rendra l’expérience inutilisable.
+
+Pour cette raison, les équipes de développement ont la possibilité de désactiver cette fonctionnalité en définissant l’une des deux propriétés suivantes dans la balise meta viewport&nbsp;:
+
+1. `user-scalable` défini à `0` ou `no`
+
+2. `maximum-scale` défini à `1`, `1.0`, etc.
+
+<figure>
+  <a href="/static/images/2019/mobile-web/fig5.png">
+    <img src="/static/images/2019/mobile-web/fig5.png" alt="Figure 5. Pourcentage de sites web de bureau et mobiles qui activent ou désactivent la possibilité de zoomer&nbsp;/&nbsp;la mise à l’échelle." aria-labelledby="fig5-caption" aria-describedby="fig5-description" width="600" height="370" data-width="600" data-height="370" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vQcVHQTKIULwgs3f2Jy8IQiHwVAJjKoHrfcvwYX5UAlb4s3bsEA2owiku4c14YZiJeG8H8acgSUul2N/pubchart?oid=655301645&amp;format=interactive">
```

Comment:
Not sure to be honest. It could be displayed in search engines (SEO is the only reason we add the Alt text to be honest as the `aria-labelledby` is used for AT devices).

Also would be displayed if the image failed to load, so again you'd probably want the `nbsp;` then.

Think I'd say leave the `&nbsp;` in, but not that fussed either way to be honest.

Reply:
I agree with @bazzadp